app/services/refresh_materialized_views.rb

```diff
@@ -18,8 +18,17 @@ def logger
     @logger ||= Rails.logger.child(class: self.class.name)
   end
 
+  def benchmark_and_statsd(operation)
```

Comment:
Is it an operation? Maybe we could call it "view"?

Reply:
Yeah, its slightly more general than just the view name because we also capture the timings of the entire run for all views.